### PR TITLE
Remove product update modal

### LINKED ISSUE #5771: login experience on gitcoin needs patching
this login experience is ridiculously bad

screencap - https://bits.owocki.com/p9u5OJlx

two modals show up and the cookie warning

then when you click out of the product updates modal; you cant actually click out of anything else.

expected result
1. no more bounty experience pop up upon first login
2. no more 'product update' pop up upon login

### PR DESCRIPTION
**Pull request by iamonuwa**
##### Description
Remove popups on login

##### Refers/Fixes
Fixes #5771 

##### Testing

clear localstorage and login again. it won't show again.

### CODE REVIEW

File: app/dashboard/templates/dashboard/index.html [hunk 1]
```diff
@@ -177,7 +177,6 @@ <h3 class="popover-header"></h3>
     </script>
     <script src="/dynamic/js/tokens_dynamic.js"></script>
     <script src="{% static "v2/js/tokens.js" %}"></script>
-    <script src="{% static "v2/js/capture-rating.js" %}"></script>
```
Comment (owocki) [inline app/dashboard/templates/dashboard/index.html]: is this going to break capturing of ratings on the profile page/rating tab, or in the rating flow on the bounty experience?

Comment (iamonuwa) [reply app/dashboard/templates/dashboard/index.html]: The profile rating is different from this

File: app/assets/v2/js/base.js [hunk 2]
```diff
@@ -403,10 +403,6 @@ const gitcoinUpdates = (force) => {
   });
 };
 
-if (document.contxt.github_handle) {
-  gitcoinUpdates();
-}
```
Comment (thelostone-mc) [inline app/assets/v2/js/base.js]: ^ can we update the `gitcoinUpdates` to remove the `force` argument + update it's invocation in `footer.html`

Comment (iamonuwa) [reply app/assets/v2/js/base.js]: We can do that.

Comment (iamonuwa) [reply app/assets/v2/js/base.js]: Removing `force` will not stop the popups. Hence I had to remove it entirely.

Comment (thelostone-mc) [reply app/assets/v2/js/base.js]: I meant we remove this and update the function so that it no longer needs an argument `force`

Comment (iamonuwa) [reply app/assets/v2/js/base.js]: Without the force, the popup will still be there

Comment (thelostone-mc) [reply app/assets/v2/js/base.js]: Ah it doesn't get invoked elsewhere ! worth removing the entire function itself?

Comment (iamonuwa) [reply app/assets/v2/js/base.js]: Noted.

Comment (iamonuwa) [reply app/assets/v2/js/base.js]: Its called from the footer as well as on login. I've removed the login call. Except you need me to remove the entire function and the link on footer

Comment (thelostone-mc) [reply app/assets/v2/js/base.js]: @iamonuwa updated the code to show what i meant !
Removed unused code in the function

**Pull request merged**